scripts/xr-compose

```diff
@@ -1596,7 +1596,12 @@ class XRCompose:
         )
         service_yaml = self.yaml["services"][service.service_name]
 
-        if self.privileged:
+        container_priv = (
+            service_yaml["privileged"]
+            if "privileged" in service_yaml.keys()
+            else self.privileged
+        )
+        if container_priv:
```

Comment:
I think this can be simpler:
```suggestion
        if "privileged" not in service_yaml and self.privileged:
```

Reply:
But someone could have privileged = False in the yaml file. Basically a situation where someone wants all expect one container to be privileged

Reply:
Sorry yes hadn't fully understand the logic you'd proposed.

I was thinking if `privileged: False` is explicitly set in the yaml file then this should take precedence, although CLI args would normally take precedence - it's just a bit of an unusual case where the CLI arg applies more widely, i.e. to all containers.

I think given
`  -i IMAGE, --image IMAGE  Name/ID of loaded XRd image to launch. This will be overridden by any images specified in the input YAML.`
I'd prefer the `--privileged` behaviour to match this, i.e. what I proposed (plus update the help output).